crates/graphql/src/schema.rs

```diff
@@ -5,13 +5,92 @@ use dataloader::{non_cached::Loader, BatchFn};
 use indexer_core::{
     db::{
         models,
-        tables::{metadata_creators, metadata_jsons, metadatas, storefronts},
+        tables::{attributes, metadata_creators, metadata_jsons, metadatas, storefronts},
         Pool,
     },
     prelude::*,
 };
 use juniper::{EmptyMutation, EmptySubscription, GraphQLObject, RootNode};
 
+#[derive(Debug, Clone)]
+struct Creator {
+    address: String,
+}
+
+#[juniper::graphql_object(Context = AppContext)]
+impl Creator {
+    fn address(&self) -> String {
+        self.address.clone()
+    }
+
+    pub fn attribute_groups(&self, context: &AppContext) -> Vec<Property> {
+        let conn = context.db_pool.get().unwrap();
+
+        let metadatas: Vec<String> = metadata_creators::table
+            .select(metadata_creators::metadata_address)
+            .filter(metadata_creators::creator_address.eq(self.address.clone()))
```

Comment:
This is a nifty feature of graphql is using the context of the parent node in downstream lookups. very powerful.